geode-assembly/src/acceptanceTest/resources/gradle-test-projects/management/build.gradle

```diff
@@ -24,7 +24,7 @@ repositories {
 }
 
 dependencies {
-  compile("${project.group}:geode-core:${project.version}")
+  compileOnly("${project.group}:geode-core:${project.version}")
```

Comment:
On my Mac, if I change this to `implementation`, the `GradleBuildWithGeodeCoreAcceptanceTest` completes without error on both JDK 8 and JDK 11. I suspect this change will also fix on Windows assembly test failure.